### PR TITLE
Update release teaser to version 14.6.0

### PR DESCRIPTION
**Pull request by HDinger**
https://community.openproject.org/projects/openproject/work_packages/57236/activity

### CODE REVIEW

File: config/locales/js-en.yml [hunk 1]
```diff
@@ -399,16 +399,17 @@ en:
           learn_about: "Learn more about the new features"
           # Include the version to invalidate outdated translations in other locales.
           # Otherwise, e.g. chinese might still have the translations for 10.0 in the 12.0 release.
-          "14_5":
+          "14_6":
             standard:
               new_features_html: >
-                The release contains various new features and improvements, such as: <br>
+                The release contains various new features and improvements, e.g. <br>
```
Comment (bsatarnejad) [inline config/locales/js-en.yml]: Based on the specification, It should be:

> The release brings various features and improvements for you, e.g.

File: frontend/src/app/features/homescreen/blocks/new-features.component.ts [hunk 2]
```diff
@@ -33,12 +33,12 @@ import { I18nService } from 'core-app/core/i18n/i18n.service';
 import { imagePath } from 'core-app/shared/helpers/images/path-helper';
 
 // The key used in the I18n files to distinguish between versions.
-const OpVersionI18n = '14_5';
+const OpVersionI18n = '14_6';
 
-const OpReleaseURL = 'https://www.openproject.org/docs/release-notes/14-5-0/';
+const OpReleaseURL = 'https://www.openproject.org/docs/release-notes/14-6-0/';
```
Comment (bsatarnejad) [inline frontend/src/app/features/homescreen/blocks/new-features.component.ts:38]: Receive 404 error, is it ok? It will be solved later?

Comment (corinnaguenther) [reply frontend/src/app/features/homescreen/blocks/new-features.component.ts:38]: Yes, this link should be available together with the release. I checked the last teaser PR, it uses the same URL style: https://github.com/opf/openproject/pull/16588/commits/d1bffeba22fa0a045e0908ebc7c15ea5933e73c9

File: config/locales/js-en.yml (hunk 1 above)
Comment (corinnaguenther) [reply config/locales/js-en.yml]: Thank you, that's right, we slightly changed the wording so that it fits the style of the bullet points below.

**Review by bsatarnejad — APPROVED** [review]

**Pull request merged**